.evergreen/config.yml

```diff
@@ -817,7 +817,8 @@ tasks:
 
     - name: "release"
       tags: ["release"]
-      git_tag_only: true
+      exec_timeout_secs: 216000  # 60 minutes (manylinux task is slow).
+      git_tag_only: false
```

Comment:
Is this going to get flipped back to true before merging?

Reply:
Done.